applications.json

```diff
@@ -3420,6 +3420,23 @@
                 "swift"
             ]
         },
+        {
+            "short_description": "A software attempt to address the \"double key press\" issue on Apple's butterfly keyboard ",
+            "categories": [
+                "keyboard"
+            ],
+            "repo_url": "https://github.com/aahung/Unshaky",
+            "title": "Unshaky",
+            "icon_url": "",
+            "screenshots": [
+                "https://github.com/aahung/Unshaky/blob/master/README/menubar.png?raw=true",
+                "https://github.com/aahung/Unshaky/blob/master/README/how-unshaky-works.png?raw=true"
```

Comment:
@blombard Please remove "?raw=true" from url's to prevent redirection.

Reply:
Sorry ! Just did it :)